library/src/main/java/com/joom/grip/io/IoFactory.kt

```diff
@@ -16,35 +16,43 @@
 
 package com.joom.grip.io
 
-import java.io.File
+import java.nio.file.Path
+import kotlin.io.path.exists
+import kotlin.io.path.extension
+import kotlin.io.path.isDirectory
 
 object IoFactory : FileSource.Factory, FileSink.Factory {
-  override fun createFileSource(inputFile: File): FileSource {
-    return when (inputFile.fileType) {
-      FileType.EMPTY -> EmptyFileSource
-      FileType.DIRECTORY -> DirectoryFileSource(inputFile)
-      FileType.JAR -> JarFileSource(inputFile)
+
+  override fun createFileSource(inputPath: Path): FileSource {
+    return when (inputPath.sourceType) {
+      SourceType.EMPTY -> EmptyFileSource
+      SourceType.DIRECTORY -> DirectoryFileSource(inputPath)
+      SourceType.JAR -> JarFileSource(inputPath)
+      SourceType.JRT -> JrtFileSource(inputPath)
     }
   }
 
-  override fun createFileSink(inputFile: File, outputFile: File): FileSink {
-    return when (inputFile.fileType) {
-      FileType.EMPTY -> EmptyFileSink
-      FileType.DIRECTORY -> DirectoryFileSink(outputFile)
-      FileType.JAR -> JarFileSink(outputFile)
+  override fun createFileSink(input: Path, output: Path): FileSink {
```

Comment:
Why is it `inputPath` in `createFileSource`, but just `input` and `output` here?

Reply:
renamed to `inputPath` and `outputPath`

Reply:
I don't see this change.